kitsune/users/jinja2/users/includes/login-fxa.html

```diff
@@ -2,10 +2,14 @@
   <div class="sumo-page-section">
     <h1 class="sumo-page-heading">{{ _('Log in / Sign up with Firefox Accounts') }}</h1>
     <p>
-      {{ _('We upgraded our log-in system to replace Mozilla Support accounts with Firefox Accounts. Firefox Accounts
-      offers one log-in for convenient and secure access to Mozilla services.') }}
-      <a href="/kb/firefox-accounts-mozilla-support-faq"> {{ _('See our FAQ') }}</a>
-      {{ _('for more information') }}.
+      {% if request.session.get('in-aaq') %}
```

Comment:
I assume this is all in a separate file to auth.html for historical reasons, but I don't see any need now: auth.html is the only other template to reference this one.

Even without merging it with auth.html, we should still use the same `aaq_product` variable as we use there.